docs/design/keyboard-shortcuts.md

```diff
@@ -1,314 +1,473 @@
+
+# [Unified app manifest for Microsoft 365](#tab/jsonmanifest)
 
 > [!NOTE]
-> To start with a working version of an add-in with keyboard shortcuts already enabled, clone and run the sample [Excel Keyboard Shortcuts](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-keyboard-shortcuts). When you're ready to add keyboard shortcuts to your own add-in, continue with this article.
+> Implementing keyboard shortcuts with the unified app manifest for Microsoft 365 is in public developer preview. It's currently only available in **Excel**. This shouldn't be used in production add-ins. We invite you to try it out in test or development environments. For more information, see the [Public developer preview app manifest schema](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview).
 
-There are three steps to add keyboard shortcuts to an add-in.
+1. In your add-in project, open the **manifest.json** file.
+1. Add the following object to the "extensions.runtimes" array. Note the following about this markup.
 
-1. [Configure the add-in's manifest](#configure-the-manifest).
-1. [Create or edit the shortcuts JSON file](#create-or-edit-the-shortcuts-json-file) to define actions and their keyboard shortcuts.
-1. [Add one or more runtime calls](#create-a-mapping-of-actions-to-their-functions) of the [Office.actions.associate](/javascript/api/office/office.actions#office-office-actions-associate-member) API to map a function to each action.
+    - The SharedRuntime 1.1 requirement set is specified in the "requirements.capabilities" object. This is required to support custom keyboard shortcuts.
+    - The "actions" objects specify the functions your add-in can run. In the following example, an add-in will be able to show and hide a task pane. You'll create these functions in a later section. Currently, custom keyboard shortcuts can only run actions that are of type "executeFunction".
+    - While the "actions.displayName" property is optional, it's required if a custom keyboard shortcut will be created for the action. This property is used to describe the action of a keyboard shortcut. The description you provide appears in the dialog that's shown to a user when there's a shortcut conflict between multiple add-ins or with Microsoft 365. It's a good practice to include the name of the add-in in the property, so that the user can easily identify to which add-in the action belongs. For more information on how conflicts with keyboard shortcuts are handled, see [Avoid key combinations in use by other add-ins](#avoid-key-combinations-in-use-by-other-add-ins).
 
-## Configure the manifest
+    ```json
+    "runtimes": [
+        {
+            "requirements": {
+                "capabilities": [
+                    {
+                        "name": "SharedRuntime",
+                        "minVersion": "1.1"
+                    }
+                ]
+            },
```

Comment:
We should delete this "requirements" property. I'll explain in email.

Reply:
Deleted.